app/presenters/addresses_presenter.rb

```diff
@@ -0,0 +1,10 @@
+class AddressesPresenter
```

Comment:
don't understand why do you need this presenter
it contains only one strange method `addresses`
which doesn't return addresses